src/components/ReleaseMonitor/ReleaseListHeader.tsx

```diff
@@ -0,0 +1,35 @@
+import { createTableHeaders } from '../../shared/components/table/utils';
+
+export const releaseTableColumnClasses = {
+  name: 'pf-m-width-20 ',
+  status: 'pf-m-width-20',
+  completionTime: 'pf-m-width-20',
+  component: 'pf-m-width-20',
+  application: 'pf-m-width-20',
+  releasePlan: 'pf-m-width-20',
+  namespace: 'pf-m-width-20 ',
+};
+
+export const applicationTableColumnClasses = {
```

Comment:
it seems we're not using this object anywhere in the code changes 🤔 can you check and remove it if not used?

Reply:
Oh, thanks, I removed it.